interactions:
- request:
    body: null
    headers:
      Accept:
      - application/json
      Accept-Encoding:
      - gzip, deflate
      CommandName:
      - network private-dns link vnet show
      Connection:
      - keep-alive
      ParameterSetName:
      - -g -n -z
      User-Agent:
      - python/3.7.4 (Windows-10-10.0.18362-SP0) msrest/0.6.10 msrest_azure/0.6.2
        azure-mgmt-privatedns/0.1.0 Azure-SDK-For-Python AZURECLI/2.0.73
      accept-language:
      - en-US
    method: GET
    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest_privatedns000001/providers/Microsoft.Network/privateDnsZones/clitest.privatedns.com000002/virtualNetworkLinks/clitestprivatednslink000003?api-version=2018-09-01
  response:
    body:
      string: '{"error":{"code":"ResourceNotFound","message":"The Resource ''Microsoft.Network/privateDnsZones/clitest.privatedns.com000002/virtualNetworkLinks/clitestprivatednslink000003''
        under resource group ''clitest_privatedns000001'' was not found."}}'
    headers:
      cache-control:
      - no-cache
      content-length:
      - '304'
      content-type:
      - application/json; charset=utf-8
      date:
      - Thu, 19 Sep 2019 23:32:50 GMT
      expires:
      - '-1'
      pragma:
      - no-cache
      strict-transport-security:
      - max-age=31536000; includeSubDomains
      x-content-type-options:
      - nosniff
      x-ms-failure-cause:
      - gateway
    status:
      code: 404
      message: Not Found
version: 1
